### PR TITLE
Add blog post for android release 3.0.0

### PR DESCRIPTION
**Pull request by dshokouhi**
## Proposed change

We are getting ready for a new android release this week so I have prepared a blog post to get reviewed so we can release with the app.

## Type of change

None of the below its a blog post :)
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

I will be creating a PR shortly to remove the `BETA` flags from the companion site so we can remove them when this PR gets merged https://github.com/home-assistant/companion.home-assistant/pull/382

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by dshokouhi** [comment]
I will make sure to update the date once we get closer to deciding upon it.

**Comment by dshokouhi** [comment]
CC: @JBassett

**Comment by dshokouhi** [comment]
Leaving as a draft so it doesn't get accidentally merged :)

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
Hey everyone its been a little while since we last spoke. We have a brand new release that we are excited to get into your hands and will be available in the today in the Google Play store.
```

Not sure it made sense before?

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
Before we get into the release details I'd like to mention that this month marks the [1 year anniversary](https://youtu.be/tc17q1Zn0Xs?t=3487) of the Home Assistant Companion app being offered in the Google Play Store! Just to think how far along the entire app has come from its initial alpha state to what is currently being offered in today's release. Can you believe its already been a year?
```

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
It has been pretty busy over on the Android side of things during Hacktoberfest! We have seen a total of 84 pull requests from a wide range of contributors that we are thankful for, so thank you for all of your efforts! In case you missed it we actually had a release shortly after version 2.4.0 but as we did not have an accompanying blog post, we'll cover whats new since we last spoke in this post. All of the features listed below are already documented on the [Companion](https://companion.home-assistant.io/) site, please be sure to check in there for greater details. Before we continue with what's new lets go over the breaking changes for this release.
```

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
In out [last blog post](https://www.home-assistant.io/blog/2020/09/12/android-240-release/) we mentioned that we were going to be breaking up some sensors and their attributes. As of 3.0.0 we have completed this task and no longer have non-static attributes as part of any sensor. If you are missing any data please check the Manage Sensors screen under App Configuration and enable the sensor you were previously using as an attribute.
```

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
<img src='/images/blog/2020-11-05-android-300-release/onboarding.png' alt='Screenshot of the new onboarding flow'></a>
Screenshot of the new onboarding flow.
```

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]: I think these look odd with the number at the start. Maybe drop them? It should be easy to tell how many there are from reading the items

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]: Link to KeyGaurd description would be good

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
- Last Notification sensor - A very powerful sensor that requires a special permission you must grant to read all notifications posted on the device. All attributes of the notification are provided as attributes for the sensor. You can think of this sensor as a great way to integrate any app that posts a notification to your device allowing you to automate around it. Personally I have been using it to integrate a food delivery app, to detect when my order will be delivered and automate around it. This sensor will update as soon as a notification is posted.
```

This feature is AWESOME! The example didn't quite seem to come across clearly to me, maybe this wording is better maybe I've got the wrong end of the stick?

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]: Also probably want to drop the location part of the URL

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
- Last Notification - This sensor also has an allow list to let the user create a list of apps from which they want notification data from. By default all notifications are sent to your Home Assistant server. We highly recommend that you set up an allow list as soon as you can think of one to prevent a high amount of updates. You will be surprised by the amount of data that will show up in a short time period.
```

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
- Location Sensors - All 3 location sensors now have settings to allow you to adjust the minimum accuracy required to send a update to your Home Assistant server. There is also a setting to adjust the minimum amount of time between updates. This should help a lot of users out who don't get the location results they expect. We recommend changing this setting after you evaluate all of the location fixes in 3.0.0 as location tracking may already be improved without needing to adjust these.
```

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
- WiFi BSSID - This sensor has a setting that will allow the user to provide an alias for the currently connected BSSID. Not everyone can remember a MAC address let alone dozens of them. This setting is designed to help out those who use this sensor to make better sense of things without the need for secrets or templates. If you live in a household with multiple access points you may find it useful to set up an alias to help with things like room presence. By default this sensor reports the connected MAC address.
```
Renaming a BSSID could be seen to be actually changing it. Nice feature too.

"users who use" just sounds odd

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
- App language can now be overridden to match a user's profile, this will impact the name of entities that get added as well as the App Configuration appearance. By default your device will use your phones language however, we noticed some users actually prefer Home Assistant to always be in a different language to their device. With this feature you can now set the language to be one of any that are supported by the app.
```

Just to remove the English bias. A link to Lokalize and a "want to help translate the app" bit here might be a good idea.

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [inline source/_posts/2020-11-12-android-300-release.markdown]: I think these need expanding or making links to the docs

**Review by TomBrien — CHANGES_REQUESTED** [review]
I think more links to the specific pages in the docs would really help. There some really sweet features here!

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: The reason why I said in the next day is because of the rollout schedule for Google Play Store. Not all users get it right away.

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: I think what you have reads better, I was sorta repeating what I said in the previous sentence :)

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: Will remove and link to the docs for each of these :)

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: alias is much better, thank you :)

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: Expanded on a few and linked to the docs on the ones we have documented :)

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: Great suggestion!

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (TomBrien) [reply source/_posts/2020-11-12-android-300-release.markdown]: Yeah I figured that might be it. "and will roll out over the next day or so"?

File: source/_posts/2020-11-12-android-300-release.markdown
Comment (agners) [inline source/_posts/2020-11-12-android-300-release.markdown]:
```suggestion
- [Traffic Stat sensors](https://companion.home-assistant.io/docs/core/sensors#traffic-stats-sensor) - Sensors whose state represent the amount of data sent and received by the device. Mobile data may not be accurate, it depends on the data that we get from the API.
```

Comment (dshokouhi) [reply source/_posts/2020-11-12-android-300-release.markdown]: Ah oops, thank you!!

**Comment by dshokouhi** [comment]
The release has deployed! Once we get approval here we are good to publish!

**Comment by dshokouhi** [comment]
I may have confused the bot by linking to a PR in the companion docs which is to remove the beta tags from this release :)

**Comment by TomBrien** [comment]
Markdown file name is 2020-11-12?

**Review by JBassett — APPROVED** [review]

**Pull request merged**
